native/src/main/java/module-info.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+ module io.ballerina.lib.sap {
+     requires sapjco3;
+     requires sapidoc3;
+    requires io.ballerina.runtime;
+     requires org.slf4j;
+     requires java.logging;
```

Comment:
Why do we need both slf4j and java.logging?

Reply:
Changed with [3b98cc3](https://github.com/ballerina-platform/module-ballerinax-sap.jco/pull/1/commits/3b98cc39afaeb4f6ad60a496009006ee83f96d96).